app/api/v1/views.py

```diff
@@ -0,0 +1,92 @@
+from flask import Flask, make_response, jsonify,request
+from flask_restful import Resource, reqparse
+from flask_jwt_extended import (jwt_required, get_jwt_identity)
+
+
+from app.api.v1.models import Product as p, products
+from app.api.v1.models import Sales as s, sales
```

Comment:
avoid using single letter variable names like p, s, U, e.t.c. Use more descriptive variable names that make sense.

Reply:
Thank you, am fixing it.